apps/explorer/tests/utils/test_utils.py

```diff
@@ -0,0 +1,160 @@
+import pandas
+import yaml
+
+from ...utils import export_pixelsets
+from apps.core.models import PixelSet
+from apps.core.tests import CoreFixturesTestCase
+from apps.core import factories
+
+
+class ExportPixelsSetTestCase(CoreFixturesTestCase):
+
+    def _assert_archive_is_valid(self, zip_archive):
```

Comment:
I usually add an empty line after function/method signature to improve readability.